rho_playbook.yml

```diff
@@ -16,6 +16,7 @@
     - dmi
     - etc_release
     - file_contents
+    - fuse_on_karaf
```

Comment:
I would prefer we keep the jboss work together and call the role `jboss_fuse_on_karaf` placing it towards the end with the `jboss_eap` role

Reply:
I moved everything to the `jboss_eap` role and updated the fact_docs and `remote_programs.rst`. Thank you!